java/src/jmri/jmrit/beantable/LogixTableAction.java

```diff
@@ -997,6 +1019,29 @@ private void copyLogixPressed(String lgxName) {
         _baseEdit.addLogixEventListener(new ConditionalBaseListener(lgxName));
     }
 
+    /**
+     * Set the user name input field.
+     * @param e The action event.
+     */
+    private void copyComboListener(ActionEvent e) {
+        if (!e.getActionCommand().equals("comboBoxChanged")) {
+            return;
+        }
+
+        var name = "";
+        var index = _copyCombo.getSelectedIndex();
+        if (index > 0) {
```

Comment:
Is this correct? I think it should be `if (index != -1) {`

https://docs.oracle.com/javase/7/docs/api/javax/swing/JComboBox.html#getSelectedIndex()
```
Returns:
    an integer specifying the currently selected list item,
    where 0 specifies the first item in the list;
    or -1 if no item is selected or if the currently selected item is not in the list
```

Reply:
The first row of the combo list is for entering a new Logix system name.  The remaining rows are the system names for existing Logix definitions.  When row 0 is selected the user name text field is set empty by skipping the if statement.  When an existing Logix is selected, its user name is copied to the user name text field.

When the combo list is created, the selected index is set to 0.

Reply:
I see. Thanks for explaining it.